### ISSUE TITLE
Ionic Lab display seems off -- device simulators do not vertically fit, scrollbars are visible 

### ISSUE THREAD

**Issue by bnfrnz** [ionic-team/ionic-cli#3658]
Ever since upgrading to `ionic-cli@^4` and installing `@ionic/lab`, when developing on a 13 inch Mac Air in Chrome (with bookmarks bar visible), the device simulators do not vertically fit anymore. There are constant (ugly white) scrollbars. 

It's a bit frustrating because:
1. The simulators are never fully visible.
2. I always scroll the lab UI instead of the app content in the device simulators, depending on where the mouse cursor currently is.

I'm not sure what changed from v3 actually. Did the device simulators' height/size increase? Did the lab header and/or footer height grow and take up more space? Something happened. And the experience, developing in a browser on a 13 inch screen has become quite annoying.

**Comment by bnfrnz** [comment]
Following [this](https://github.com/ionic-team/ionic-cli/issues/3528#issuecomment-424983630), opening `http://localhost:8100/ionic-lab`, the old lab UI seems to be used and everything fits perfectly again.

**Comment by dwieeb** [comment]
@bnfrnz Ionic Lab was completely rewritten so it could be reusable. I will investigate this.

**Comment by dwieeb** [comment]
@bnfrnz Not really seeing any of the issues you're describing, unless I'm missing something. In fact, the new Ionic Lab seems more usable. 

New:

* I can scroll up and down to see the top and bottom of the devices.

<img width="1198" alt="image" src="https://user-images.githubusercontent.com/236501/46698071-c220b580-cbdb-11e8-8c36-5ba02cbeca39.png">

Old:

* Won't let me scroll to reveal the status bars.

<img width="1201" alt="image" src="https://user-images.githubusercontent.com/236501/46698152-f2685400-cbdb-11e8-9f96-1ae029e346a5.png">

**Comment by dwieeb** [comment]
We _could_ shrink the devices so that they fit in the screen, but I don't know if that's always desirable. Maybe we can add a setting.

**Comment by bnfrnz** [comment]
Hi @dwieeb, thanks for looking into this.

> Won't let me scroll to reveal the status bars.

In your screenshot of the old Ionic Lab, your window height is very small. I don't think anyone uses a screen like that for development. I was referring to at least a 13 inch MacBook. In that case, you will not need any scrollbars in the old Lab. Everything is just perfectly visible.

To have everything visible at once in the new Lab, you'd need a 15 inch laptop. I opened this issue because I often work on 13 inch. And I'd assume there are a fair few people out there who do as well.

Here are 13 inch screenshots:

**Old Lab:**
![old](https://user-images.githubusercontent.com/26401091/46711982-66106c80-cc81-11e8-8c63-15fffbdc98d6.png)

**New Lab:**
![new](https://user-images.githubusercontent.com/26401091/46711998-76284c00-cc81-11e8-909f-b29526a78f38.png)

**I think these are the main changes from old to new:**
![new with markup](https://user-images.githubusercontent.com/26401091/46712012-83453b00-cc81-11e8-8533-d196d582f831.png)


> I can scroll up and down to see the top and bottom of the devices.

As said, in my opinion scrolling is not desirable here. Because my app (within the device iframes) scrolls as well. So, when I use a Magic Mouse (or any conventional mouse with a wheel) to scroll up and down, I just want my app to scroll. It's a bit frustrating when the entire Lab window alway scrolls up and down instead of just my app.

I hope I did a better job at explaining this time but let me know if not...

> We could shrink the devices so that they fit in the screen, but I don't know if that's always desirable. Maybe we can add a setting.

I think a setting like that could be a good solution! It would enable people with different screen sizes to develop and test without having to scroll up and down constantly.

I did try using the browser-inbuilt zoom function for this. But that gives some weird results, especially when compressing image assets.
